frontend/src/routes/data/CustomData/AuditTypeList.tsx

```
@@ -0,0 +1,339 @@
+                        disabled={isDisabled}
+                        id="name"
+                        name="name"
+                        onChange={e => handleInputChange(row.name, 'name', e)}
+                        placeholder={t('name')}
+                        type="text"
+                        value={row.name}
+                      />
+                    </div>
+                    <div className="text-lg">{t('auditPhase')}</div>
+                    <DefaultRadioGroup
+                      name={'stage' + row.name}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)